packages/ffe-checkbox-react/src/Checkbox.js

```diff
@@ -24,6 +32,8 @@ export default function CheckBox(props) {
                     'ffe-checkbox--no-margin': noMargins,
                 })}
                 htmlFor={id}
+                role="presentation"
+                onClick={ event => { if (stopPropagation) { event.stopPropagation() } } }
```

Comment:
It runs just fine. Clicking the `label` also triggers a click event on the `input`. Not because of event bubbling - these are two separate events triggered because of the link between them through `id` and `htmlFor`.

I tried writing a test for this, but clicking the `label` with `enzyme` doesn't trigger the event on the `input`. This is simply a limitiation of `enzyme`.